CommunityToolkit.Mvvm/Input/AsyncRelayCommand.cs

```diff
@@ -323,4 +330,13 @@ public void Cancel()
             PropertyChanged?.Invoke(this, IsCancellationRequestedChangedEventArgs);
         }
     }
+
+    /// <summary>
+    /// Awaits an input <see cref="Task"/> and throws an exception on the calling context, if the task fails.
+    /// </summary>
+    /// <param name="executionTask">The input <see cref="Task"/> instance to await.</param>
+    internal static async void AwaitAndThrowIfFailed(Task executionTask)
+    {
+        await executionTask;
```

Comment:
can we add the `ConfigureAwait(false)` here?
```suggestion
        await executionTask.ConfigureAwait(false);
```

Reply:
I didn't add this on purpose to be honest. The idea is that if one of the main points of having exceptions be rethrown is to also help debugging things (other than not get the app into a weird invalid state of course), I feel like it'd be clearer for users to see the exceptions be thrown on the calling context, which would generally be the UI thread. This is also handled in special ways on many frameworks, versus exceptions being thrown on random thread pool threads. I know there's a slight overhead in this, but I feel like that's fine here given it's not like this would be used in a hot path like in some asynchronous code dealing with loops of web requests or anything like that 🤔

Reply:
It's fine then. I thought that could be intentional, but not sure if with the `ConfigureAwait(false)` the exception will be thrown in another context. Thanks for the explanation (: